### PR TITLE
Edited the card color to light gray

### PR DESCRIPTION
**Pull request by NnajiChimuanya**
projectmate

Edit the card color to light gray 
This closes issue #230

### CODE REVIEW

**Comment by RakeshSangem** [comment]
We are using Tailwind  CSS know?

File: pages/user/profile.tsx [hunk 1]
```diff
@@ -74,18 +74,27 @@ const Profile: NextPage = () => {
...
+              style={{ backgroundColor: 'rgb(243 244 246)' }}
+            >
               <BsFillBookmarkDashFill />
             </button>
           </div>
           <div className="relative h-96 w-full rounded-lg shadow-border-shadow dark:bg-[#232931] sm:max-w-2xl">
-            <button className="absolute top-5 right-4 p-3 transition-all hover:opacity-70">
+            <button
+              className="absolute top-5 right-4 p-3 transition-all hover:opacity-70"
+              style={{ backgroundColor: 'rgb(243 244 246)' }}
```
Comment (rohitdasu) [inline pages/user/profile.tsx:96]: please don't use inline styles instead use tailwind classes

File: package.json [hunk 2]
```diff
@@ -8,7 +8,8 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "prisma:generate": "node ./node_modules/.bin/prisma generate --schema=./prisma/schema.prisma"
```
Comment (rohitdasu) [inline package.json:12]: what it does? could you explain

**Review by rohitdasu — CHANGES_REQUESTED** [review]

**Comment by rohitdasu** [comment]
also you've changed the color for bookmarks not for projects.

**Pull request closed without merging**